sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SdkMeterProvider.java

```diff
@@ -95,16 +95,13 @@ public static SdkMeterProviderBuilder builder() {
    *   .buildInstrument();
    *
    * // create a specification of how you want the metrics aggregated:
-   * AggregationConfiguration viewSpecification =
-   *   AggregationConfiguration.create(Aggregations.minMaxSumCount(), Temporality.DELTA);
+   * AggregationFactory aggregationFactory = AggregationFactory.minMaxSumCount();
    *
    * //register the view with the MeterSdkProvider
-   * meterProvider.registerView(instrumentSelector, viewSpecification);
+   * meterProvider.registerView(instrumentSelector, aggregationFactory);
    * }</pre>
-   *
-   * @see AggregationConfiguration
    */
-  public void registerView(InstrumentSelector selector, AggregationConfiguration specification) {
+  public void registerView(InstrumentSelector selector, AggregatorFactory specification) {
```

Comment:
this parameter name doesn't seem to make sense any more. rename to `aggregationFactory` I think.

Reply:
Done.